api/apps/api/src/modules/geo-features/geo-features.service.ts

```diff
@@ -258,4 +269,91 @@ export class GeoFeaturesService extends AppBaseService<
   ): Promise<GeoFeature> {
     return entity;
   }
+
+  public async createFeaturesForShapefile(
```

Comment:
transactions here are little bit sketchy, since I'm writing data to 2 separate DBs :thinking: if you think of a better solution let me know :smile:

Reply:
yes, it occurred to me with a huge 🤦🏼 when I saw your PR while trying to help with the makevalid bit - I think what you do here will be fine in most circumstances. if we do run into issues in a substantive way I'd think about a final check both in case of success or failure, but since I'm thinking about a grim reaper/garbage collection feature that we will need in any case to be able to invalidate stuff when users edit settings earlier on in the workflow, this - if needed at all - could be done when we implement that.